inc/thorvg.h

```diff
@@ -1016,7 +1017,7 @@ class TVG_EXPORT Picture final : public Paint
      *
      * @warning: you have responsibility to release the @p data memory if the @p copy is true
      */
-    Result load(const char* data, uint32_t size, bool copy = false) noexcept;
```

Comment:
This api must be retained for the compatibility. You can deprecate it or just leave it now.